interfaces/cython/cantera/kinetics.pyx

```diff
@@ -376,13 +376,16 @@ cdef class InterfaceKinetics(Kinetics):
             self._phase_indices[phase.name] = i
             self._phase_indices[i] = i
 
-    def advance_coverages(self, double dt):
+    def advance_coverages(self, double dt, double rtol=1e-7, double atol=1e-14,
+                          double max_step_size=0, int max_steps=0,
+                          int max_err_failures=0):
```

Comment:
I would suggest the parameter name be `max_err_test_fails` or `max_error_test_failures`